docs/env_var_reference.rst

```diff
@@ -0,0 +1,76 @@
+Environment Variables
+=====================
+CumulusCI has environment variables that are useful when CumulusCI is being run inside of web applications such as MetaCI, MetaDeploy, and Metecho.
+The following is a reference list of available environment variables that can be set.
+
+
+
+``CUMULUSCI_AUTO_DETECT`` 
+
+    Set this environment variable to autodetect branch and commit information from ``HEROKU_TEST_RUN_BRANCH`` and ``HEROKU_TEST_RUN_COMMIT_VERSION`` environment variables.
+
+
+
+``CUMULUSCI_DISABLE_REFRESH``
+
+    If present, will instruct CumulusCI to not refresh OAuth tokens for orgs.
+
+
+
+``CUMULUSCI_KEY``
+
+    An alphanumeric string used to encrypt org credentials at rest when an OS keychain is not available.
+
+
+
+``CUMULUSCI_REPO_URL``
+    
+    Used for specifying a Github Repository for CumulusCI to use when running in a CI environment.
+
+
+
+``GITHUB_APP_ID``
+
+    Your GitHub App's identifier.
+
+
+
+``GITHUB_APP_KEY``
+
+    Contents of a JSON Web Token (JWT) used to authenticate a GitHub app (see `GitHub’s docs <https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#authenticating-as-a-github-app>`_).
```

Comment:
```suggestion
    Contents of a JSON Web Token (JWT) used to authenticate a GitHub app (see `Authenticating as a GitHub App <https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#authenticating-as-a-github-app>`_) in GitHub documentation.
```
For accessibility reasons, make the hyperlinked text match the title of the thing that we're linking to.